backend/README.md

```diff
@@ -28,3 +28,22 @@ By default application will process blocks from the chains specified in `config/
 - `npm run build` - build project
 
 License: Apache-2.0
+
+## Docker "local mode"
```

Comment:
that is minor, but currently we have default mode - processing real-time blocks over the RPC and 'archive' mode, maybe we should stick to those names to avoid confustion